### PR TITLE
Add api

### PR DESCRIPTION
**Pull request by thienhv**

### CODE REVIEW

File: config/initializers/active_model_serializers.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+#   http://www.rubydoc.info/gems/active_model_serializers
+#   According to the official document ...
+#   By default ActiveModelSerializers will use the Attributes Adapter (no JSON root).
+#   But we strongly advise you to use JsonApi Adapter
+ActiveModelSerializers.config.adapter = :json_api
+# NOTE: AMS converts underscores to hyphens automatically to comply with recommendation of JSON API.
+#   See: http://jsonapi.org/recommendations/#naming
+#   The following setting disables that.
+#   It doesn't mean this is recommendation. fast_jsonapi doesn't do so, for instance.
+ActiveModel::Serializer.config.key_transform = :unaltered
```
Comment (jp-ryuji) [inline config/initializers/active_model_serializers.rb]: This is not required this time.
You can keep it though.

File: config/routes.rb [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 Rails.application.routes.draw do
   root 'home#index'
-
   post '/', to: 'home#search'
+
+  namespace :api do
+    namespace :v1 do
+      resources :developers, only: [:index]
```
Comment (jp-ryuji) [inline config/routes.rb]: My explanation seems a bit vague.
I meant that I wanted to create a `show` method.
I corrected the exercise document.

You can keep this implementation.
Can you add `show`?

File: config/initializers/active_model_serializers.rb (hunk 1 above)
Comment (thienhv) [reply config/initializers/active_model_serializers.rb]: @jp-ryuji ok (Y)

File: config/routes.rb (hunk 2 above)
Comment (thienhv) [reply config/routes.rb]: @jp-ryuji ok (Y)

File: config/initializers/active_model_serializers.rb (hunk 1 above)
Comment (thienhv) [reply config/initializers/active_model_serializers.rb]: @jp-ryuji 

If i remove it LOL

screen shot 2018-04-03 at 19 43 27

Comment (jp-ryuji) [reply config/initializers/active_model_serializers.rb]: @thienhv 
That's a correct behavior.
You should change the expected response.

**Comment by jp-ryuji** [comment]
@thienhv 
This PR is ok.
But I found a bug.
After the page is rendered, the search conditions are cleared.
This is not an expected behavior.
Can you fix this on a new PR?

**Comment by thienhv** [comment]
@jp-ryuji I fixed this bug LOL

**Pull request merged**